api_tests/users/views/test_user_detail.py

```diff
@@ -300,14 +300,16 @@ def setUp(self):
                     'middle_names': 'Malik el-Shabazz',
                     'family_name': 'X',
                     'suffix': 'Sr.',
-                    'github': 'newGithub',
-                    'scholar': 'newScholar',
-                    'personal_website': 'http://www.newpersonalwebsite.com',
-                    'twitter': 'http://www.newpersonalwebsite.com',
-                    'linkedin': 'newLinkedIn',
-                    'impactstory': 'newImpactStory',
-                    'orcid': 'newOrcid',
-                    'researcherid': 'newResearcherId',
+                },
+                'social': {
+                    'github': ['http://github.com/even_newer_github/'],
```

Comment:
This should be `'github': ['http://github.com/new_github/'],` so that it gets updated in the PATCH test below.